src/components/Flex/Flex.cy.tsx

```diff
@@ -0,0 +1,269 @@
+        cy.get('@Flex').should('have.class', 'tw-items-stretch');
+    });
+
+    it('should render with alignContent start', () => {
+        cy.mount(<FlexComponent alignContent="start" />);
+
+        cy.get(FLEX_ID).as('Flex');
+        cy.get('@Flex').should('have.class', 'tw-content-start');
+    });
+
+    it('should render with alignContent end', () => {
+        cy.mount(<FlexComponent alignContent="end" />);
+
+        cy.get(FLEX_ID).as('Flex');
+        cy.get('@Flex').should('have.class', 'tw-content-end');
+    });
+
+    it('should render with alignContent center', () => {
+        cy.mount(<FlexComponent alignContent="center" />);
+
+        cy.get(FLEX_ID).as('Flex');
+        cy.get('@Flex').should('have.class', 'tw-content-center');
+    });
+
+    it('should render with alignContent normal', () => {
+        cy.mount(<FlexComponent alignContent="normal" />);
+
+        cy.get(FLEX_ID).as('Flex');
+        cy.get('@Flex').should('have.class', 'tw-content-normal');
+    });
+
+    it('should render with alignContent between', () => {
+        cy.mount(<FlexComponent alignContent="between" />);
+
+        cy.get(FLEX_ID).as('Flex');
+        cy.get('@Flex').should('have.class', 'tw-content-between');
+    });
+
+    it('should render with alignContent around', () => {
+        cy.mount(<FlexComponent alignContent="around" />);
+
+        cy.get(FLEX_ID).as('Flex');
+        cy.get('@Flex').should('have.class', 'tw-content-around');
+    });
+
+    it('should render with alignContent evenly', () => {
+        cy.mount(<FlexComponent alignContent="evenly" />);
+
+        cy.get(FLEX_ID).as('Flex');
+        cy.get('@Flex').should('have.class', 'tw-content-evenly');
+    });
+
+    it('should render with alignContent baseline', () => {
+        cy.mount(<FlexComponent alignContent="baseline" />);
+
+        cy.get(FLEX_ID).as('Flex');
+        cy.get('@Flex').should('have.class', 'tw-content-baseline');
+    });
+
+    it('should render with alignContent stretch', () => {
+        cy.mount(<FlexComponent alignContent="stretch" />);
+
+        cy.get(FLEX_ID).as('Flex');
+        cy.get('@Flex').should('have.class', 'tw-content-stretch');
+    });
+
+    it("should render as a span using the 'as' prop", () => {
+        cy.mount(<FlexComponent as="span" />);
+
+        cy.get(`span${FLEX_ID}`).as('Flex');
+        cy.get('@Flex').should('exist');
+    });
+
+    it("should render as a span using the 'as' prop", () => {
+        cy.mount(<FlexComponent as="span" />);
+
+        cy.get(`span${FLEX_ID}`).as('Flex');
+        cy.get('@Flex').should('exist');
+    });
+
+    it('should render with bg', () => {
+        cy.mount(<FlexComponent bg="tw-bg-box-positive-strong" />);
+
+        cy.get(FLEX_ID).as('Flex');
+        cy.get('@Flex').should('have.class', 'tw-bg-box-positive-strong');
+    });
+
+    it('should render with color', () => {
+        cy.mount(<FlexComponent color="tw-bg-box-positive-inverse" />);
```

Comment:
Just for the correct usage of Alias Tokens, it should be `color="tw-text-box-positive-inverse"`